p3/forms.py

```diff
@@ -124,14 +124,15 @@ def save(self, *args, **kwargs):
 
         return talk
 
+
 class P3SubmissionAdditionalForm(P3TalkFormMixin, cforms.TalkForm):
-    duration = P3SubmissionForm.base_fields['duration']
+    duration         = P3SubmissionForm.base_fields['duration']
```

Comment:
IMHO we should all try to be PEP8-ish compliant.... I'm -0 on this style choice, as I don't think you'll be changing it all over the project so mixing up styles like this is not a great choice